docs/docs/other-css-frameworks.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Other CSS Frameworks
+---
+
+This guide introduces some other popular styling libraries that you might want to use in your project.
+
+## How these frameworks were chosen
```

Comment:
This isn't the kind of information I'd normally include in a guide. I think @marcysutton included it in the issue in an effort to describe the content she thought this page should have. (She's free to correct me if I'm wrong!)

You _could_ incorporate some of these points into more general advice on choosing a CSS library or framework. We don't really push for any one solution over another throughout the docs but we _can_ share strategies for making better framework/library choices.

Reply:
Got it! That's a great idea, I'll convert it.

Reply:
That is indeed the reason I included it–to inform whoever would do this work. Though we'll have to keep an eye out to see if frameworks are added by the community that don't meet our standards–that may be a good reason to add something explicit back in.

Thanks for the ping and for working on this!